backend/src/layer/data/swka_parser/test_data/test_1.html

```diff
@@ -1161,8 +1161,8 @@ <h1 class="mensa_fullname">Menseria Moltkestraẞe 30</h1>
 
 
                 <div class="col-4 col-medium-4 col-small-12">
-                  <div><select id="meal_type" name="meal_type" class="p-dd"
```

Comment:
did you run the tests after renaming? I do not think this should be renamed. Renaming is dangerous.

Reply:
Yes, I did, they all passed. I thought that we were going to update the database anyway, so I changed it. Clippy was complaining...

Reply:
but what you changed here is the HTML of the mensa webpage...

Reply:
I know. I fixed it now